feature-libs/smartedit/core/services/smart-edit.service.ts

```diff
@@ -83,23 +75,6 @@ export class SmartEditService {
       });
   }
 
-  /**
-   * load webApplicationInjector.js
-   */
-  /**
-   * @deprecated since 5.2
-   */
-  protected loadScript(): void {
-    this.scriptLoader.embedScript({
-      src: 'assets/webApplicationInjector.js',
-      params: undefined,
-      attributes: {
-        id: 'text/smartedit-injector',
-        'data-smartedit-allow-origin': this.config.smartEdit?.allowOrigin,
-      },
-    });
-  }
-
```

Comment:
There should be a migration comment for this removal in `docs/migration/6_0/migration-comments-members.json`